hoomd/mesh.py

```diff
@@ -53,28 +51,28 @@ class Mesh(_HOOMDBaseObject):
 
     Examples::
 
-        mesh = mesh.Mesh()
-        mesh.triangles = [[0,1,2],[0,2,3],[0,1,3],[1,2,3]]
+        mesh_obj = mesh.Mesh()
+        mesh_obj.types = ["mesh"]
+        mesh_obj.type_ids = [0,0,0,0]
+        mesh_obj.triangles = [[0,1,2],[0,2,3],[0,1,3],[1,2,3]]
 
     """
 
     def __init__(self):
 
-        param_dict = ParameterDict(size=int,
-                                   types=OnlyIf(
-                                       to_type_converter([str]),
-                                       preprocess=self._preprocess_type))
+        param_dict = ParameterDict(size=int, types=[str])
```

Comment:
I like the idea. But does putting the triangle data into `ParameterDict` keep them loggable?

Reply:
You would need to then add a loggable entry either directly upon construction or through a property. I forget these were loggable. Having read only properties though would allow logging, and I think we do something similar elsewhere but I forget where exactly.

We are sort of in a grey zone because the quantities here are user defined and don't change during the simulation (I think) so it isn't a normal logged quantity.

Reply:
Well, not here but later with the dynamical bonding updaters the triangles do change.  But I might add two additional properties to make them loggable.